src/Tribe/Aggregator/Cron.php

```diff
@@ -271,17 +271,18 @@ public function verify_child_record_creation() {
 
 		foreach ( $query->posts as $post ) {
 			$record = Tribe__Events__Aggregator__Records::instance()->get_by_post_id( $post );
-			if ( ! $record->is_schedule_time() ) {
-				continue;
-			}
 
-			if ( $record->get_child_record_by_status( 'pending' ) ) {
-				continue;
-			}
+			// if ( ! $record->is_schedule_time() ) {
+			// 	continue;
+			// }
 
+			// if ( $record->get_child_record_by_status( 'pending' ) ) {
+			// 	continue;
+			// }
 			$child = $record->create_child_record();
 			$child->queue_import();
 			$child->process_posts();
+			exit;
```

Comment:
Ohh yeah it will be removed after my testing is done on CSV. Thanks for the reminder.